.github/workflows/crux-llvm-build.yml

```diff
@@ -206,13 +206,22 @@ jobs:
           CLANG: "${{ github.workspace }}/llvm/bin/clang-withIncl"
 
       - shell: bash
-        run: .github/ci.sh bundle_crux_llvm_files
+        run: |
+          NAME="crux-llvm-${{ needs.outputs.outputs.crux-llvm-version }}-${{ matrix.os }}-x86_64"
+          echo "NAME=$NAME" >> $GITHUB_ENV
+          .github/ci.sh bundle_crux_llvm_files
         if: github.repository == 'GaloisInc/crucible'
         env:
           OS_TAG: ${{ matrix.os }}
+          VERSION: ${{ needs.outputs.outputs.crux-llvm-version }}
+
+      - if: github.event.pull_request.head.repo.fork == false
```

Comment:
I think it would be a good idea to use `- name: Create Binary Bundle (Docker?)` and `- name: Sign bundle` to introduce these steps and add a comment line next to the `if: ` indicating why we only sign bundles on the head repo and not forks.

[In general, I'm a favor of having a `name` for *all* steps that is informative about what the step is trying to accomplish, but naming them all is outside the scope of this PR.]